### ISSUE TITLE
Scrolling support for NestedScrollView ??

### ISSUE THREAD

**Issue by tuhin10** [jianghejie/XRecyclerView#14]
Can it handle RecyclerView scrolling if we put it inside NestedScrollView on Pre-Lolipop devices?

**Comment by jianghejie** [comment]
not recommended

**Comment by tuhin10** [comment]
yes your right. But I badly need it, is there any hack/work around to achieve this??

**Comment by fukemy** [comment]
the most common layout with stories need 2 recyclerview inside NestedScrollView, why not recommended?
